packages/blob-storage-manager/src/BlobStorage.ts

```diff
@@ -48,13 +48,21 @@ export abstract class BlobStorage {
     }
   }
 
-  async storeBlob(
-    chainId: number,
-    hash: string,
-    data: string
-  ): Promise<string> {
+  async removeBlob(uri: string): Promise<void> {
```

Comment:
What is the use case for removing a blob from a storage?

Reply:
@PabloCastellano  This PR proposes a change to the blob propagator to enable specifying any blob storage as the temporary storage, rather than limiting it to just the file system. Therefore, all blob storages should implement this method.